packages/paste-website/stories/TokensListFilter.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import {TokensListFilter} from '../src/components/tokens-list/TokensListFilter';
+
+export const TokensListFilterExample = (): React.ReactNode => (
+  // @ts-expect-error not passing in props
```

Comment:
suggestion: it would probably be safer if you give the component mock props so that we can still have the story typechecked for future proofing